packages/utils/src/time.ts

```diff
@@ -163,11 +162,21 @@ export const browserPerformanceTimeOrigin = ((): number | undefined => {
   // eslint-disable-next-line deprecation/deprecation
   const navigationStart = performance.timing && performance.timing.navigationStart;
   const hasNavigationStart = typeof navigationStart === 'number';
-  const navigationStartIsReliable =
-    hasNavigationStart && Math.abs(navigationStart + performance.now() - Date.now()) < threshold;
-  if (navigationStartIsReliable) {
-    _browserPerformanceTimeOriginMode = 'navigationStart';
-    return navigationStart;
+  // if navigationStart isn't available set delta to threshold so it isn't used
+  const navigationStartDelta = hasNavigationStart
+    ? Math.abs(navigationStart + performance.now() - Date.now())
```

Comment:
Please share the value of `performance.now()` and `Date.now()` when computing the deltas. Otherwise small differences could affect the comparisons and distract us when analyzing what gets actually used.